src/libcollections/btree/node.rs

```diff
@@ -200,6 +200,9 @@ struct RawItems<T> {
     tail: *const T,
 }
 
+unsafe impl<T: Send> Send for RawItems<T> { }
```

Comment:
IIIII'm not sure this is sound. RawItems is basically for draining values out of an allocation, but doesn't actually own its allocation. Sending this seems unsound?

Also RawItems is 100% internal, so I don't think it *needs* Send/Sync (only exported inside of MoveTraversalImpl, but you impl Send/Sync on that already).